### ISSUE TITLE
Make maxRoundTripDelay for LBS configurable

### ISSUE THREAD

**Issue by KrishnaIyer** [TheThingsNetwork/lorawan-stack#3548]
#### Summary

Make maxRoundTripDelay for LBS configurable

#### Why do we need this?

With https://github.com/TheThingsNetwork/lorawan-stack/pull/3547, maxRoundTripDelay is a fixed value. 

I've done this so that #3547 can land in `v3.10.3` and we can add the config later. 

#### What is already there? What do you see now?

#3547 where maxRoundTripDelay is a fixed value

#### What is missing? What do you want to see?

Add maxRoundTripDelay to LBS config

#### Environment

Target `v3.11` branch

#### How do you propose to implement this?

Add to config after #3547 is merged and `v3.10` is merged into `v3.11`.

#### How do you propose to test this?

UT

#### Can you do this yourself and submit a Pull Request?

Yup

**Comment by KrishnaIyer** [comment]
Not necessary since this doesn't break compatibility.
